config/permissions.yml

```diff
@@ -1622,6 +1622,10 @@ feature_project_switcher:
     label: 'Display a dropdown menu in global header to allow users to quickly move to other instances of the SH platform.'
     loginRequired: false
     parent: area_admin
+feature_profile_editable:
+    description: 'If enabled the user can his/her data edit'
```

Comment:
```suggestion
    description: 'If enabled the user can edit his/her data'
```

Reply:
done in [4e6dcac](https://github.com/demos-europe/demosplan-core/pull/2800/commits/4e6dcac3763f01ece3fe13722f0de9c8b68bc519)